cni/network/network_windows.go

```diff
@@ -296,7 +296,48 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig) []policy.Policy {
 	return policies
 }
 
-func addIPV6EndpointPolicy(nwInfo network.NetworkInfo) (policy.Policy, error) {
+func getEndpointPolicies(nwCfg *cni.NetworkConfig, netns string, nwInfo *network.NetworkInfo, ipconfigs []*cniTypesCurr.IPConfig) ([]policy.Policy, error) {
```

Comment:
based on the args passed to this func, it looks like if in future we want to get some other policy from nwCfg we would need to pass new set of params to this func, so potentially the number of params to this func will keep on growing. Instead of that can we create separate funcs to getIpv6EpPolicy and getLoopbackDsrEpPolicy? That way you can only pass the args needed to those respective funcs and we can create a new func for the new policy going forward

Reply:
the problem is we need to add that many dummy functions in linux..one thing i can do is create struct for all params thats passed..so in future that struct can be expanded if needed.

Reply:
what's the issue with adding multiple dummy functions in linux. I think having different functions will result into clear code